rootdir/init.shinano.pwr.rc

```diff
@@ -44,28 +44,25 @@ on boot
     chmod 664 /sys/devices/system/cpu/cpu2/online
     chmod 664 /sys/devices/system/cpu/cpu3/online
 
-on property:service.bootanim.exit=1
+on property:sys.boot_completed=1
```

Comment:
KK we want to run a service on a init blob that can't be edited, so we duplicate the start of that service using a different condition.

Here we can edit the init script, so I don't see any reason to make it more complicated than changing one line.